README.rst

```diff
@@ -59,7 +64,11 @@ Default HA strategy of ldap3.ServerPool is employed, so first available server i
         #bind_dn:
         #bind_password:
         #filter: "(objectClass=posixAccount)"
-        #validate_cert: true
+        #tls_options:
```

Comment:
I guess in this case if you had multiple servers with separate certs it is on you to figure out how to mash those together?

Reply:
I suppose this could be refactored to be:

```
servers:
  url:
  tls_options:
  ...
```

but tbh, as you say, let's leave this to the admin. It feels like an over-optimisation to at this time.